services/github/github-issues.tester.js

```diff
@@ -12,20 +12,77 @@ t.create('GitHub closed pull requests')
     ),
   })
 
+t.create('GitHub closed pull requests excluding drafts')
+  .get('/issues-pr-closed/badges/shields.json?excludeDrafts=true')
```

Comment:
You don't need to explicitly add a true value, including the query param will assume true value.
```suggestion
  .get('/issues-pr-closed/badges/shields.json?excludeDrafts')
```